tasks/contactdefinitive/app/src/main/java/com/example/contactdefinitive/ActivitySecond.kt

```diff
@@ -0,0 +1,63 @@
+package com.example.contactdefinitive
+
+import android.os.Bundle
+import android.view.View
+import android.widget.Toast
+import androidx.appcompat.app.AppCompatActivity
+import com.example.contactdefinitive.databinding.ActivitySecondBinding
+
+class ListContactActivity : AppCompatActivity() {
+    private lateinit var binding: ActivitySecondBinding
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_second)
+        setupListeners()
+    }
+
+    private fun setupListeners() {
+        binding = ActivitySecondBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        val bundle = intent.getBundleExtra("bundle")
+        val contactList = bundle?.getParcelableArrayList<Contact>("contact")
+
+        when (contactList != null) {
+            (contactList?.size == 1) -> with(binding) {
+                nomeDoCtt1.text = contactList!![0].name
```

Comment:
don't use non null assertion. it can cause an app crash. Call me when doing it and I will explain better